src/apis/avm/api.ts

```diff
@@ -1278,16 +1291,83 @@ export class AVMAPI extends JRPCAPI {
     } else {
       amnt = amount;
     }
+    if(typeof memo !== 'string') {
+      m = bintools.cb58Encode(memo);
+    } else {
+      m = memo;
+    }
 
     const params:any = {
-      username,
-      password,
+      username: username,
+      password: password,
       assetID: asset,
       amount: amnt.toString(10),
-      to,
-      from,
+      to: to,
+      from: from,
+      changeAddr: changeAddr,
+      memo: m
+    };
+    return this.callMethod('avm.send', params).then((response:RequestResponseData) => response.data.result);
+  };
+
+  /**
+   * Sends an amount of assetID to an array of specified addresses from a list of owned of addresses.
+   *
+   * @param username The user that owns the private keys associated with the `from` addresses
+   * @param password The password unlocking the user
+   * @param sendOutputs The array of SendOutputs. A SendOutput is an object literal which contains an assetID, amount, and to.
+   * @param from An array of addresses managed by the node's keystore for this blockchain which will fund this transaction
+   * @param changeAddr An address to send the change
+   * @param memo Optional CB58 Buffer or String which contains arbitrary bytes, up to 256 bytes
+   *
+   * @returns Promise for the string representing the transaction's ID.
+   */
+  sendMultiple = async (username:string, password:string, sendOutputs:Array<{assetID:string | Buffer, amount:number | BN, to:string}>, from:Array<string> | Array<Buffer>, changeAddr:string, memo:string | Buffer = undefined):Promise<{txID: string, changeAddr: string}> => {
+    let asset:string;
+    let amnt:BN;
+    let m:string;
+    let sOutputs:Array<{assetID:string, amount:string, to:string}> = [];
+
+    if (typeof this.parseAddress(changeAddr) === 'undefined') {
+      /* istanbul ignore next */
+      throw new Error(`Error - AVMAPI.sendMultiple: Invalid address format ${changeAddr}`);
+    }
+
+    from = this._cleanAddressArray(from, 'sendMultiple');
+
+    sendOutputs.forEach((output) => {
+      if (typeof this.parseAddress(output.to) === 'undefined') {
+        /* istanbul ignore next */
+        throw new Error(`Error - AVMAPI.sendMultiple: Invalid address format ${output.to}`);
+      }
+      if (typeof output.assetID !== 'string') {
+        asset = bintools.cb58Encode(output.assetID);
+      } else {
+        asset = output.assetID;
+      }
+      if (typeof output.amount === 'number') {
+        amnt = new BN(output.amount);
+      } else {
+        amnt = output.amount;
+      }
+      sOutputs.push({to: output.to, assetID: asset, amount: amnt.toString(10)})
+    })
+
+    if(typeof memo !== 'string') {
+      m = bintools.cb58Encode(memo);
+    } else {
+      m = memo;
+    }
+
+    const params:any = {
+      username: username,
+      password: password,
+      outputs: sOutputs,
+      from: from,
+      changeAddr: changeAddr,
```

Comment:
we cannot add changeAddr and memo by default. They should only be added to params if they need to be added (as in, they're not undefined)